scaling_performance/optimizing_storage.adoc

```diff
@@ -175,13 +175,58 @@ to nodes to support a healthy cluster.
 == Choosing a Docker Graph Driver
 
 Docker stores images and containers in a graph driver (a pluggable storage
-technology), such as `DeviceMapper`, `Overlay`, and `Btrfs`. Each have advantages and
-disadvantages. For example, `Overlay` is faster than `DeviceMapper` at starting and
-stopping containers, but is not Portable Operating System Interface for Unix (POSIX) compliant because of the architectural
-limitations of a union file system, and does not yet support SELinux.
+technology), such as `DeviceMapper`, `Overlay`, and `Btrfs`. Each have
+advantages and disadvantages. For example, `Overlay` is faster than
+`DeviceMapper` at starting and stopping containers, but is not Portable
+Operating System Interface for Unix (POSIX) compliant because of the
+architectural limitations of a union file system, and does not yet support
+SELinux.
```

Comment:
this paragraph is a summary of what's in the table. keep the first 2 sentences only (remove reference to btrfs).